app.json

```diff
@@ -25,7 +25,7 @@
       "**/*"
     ],
     "ios": {
-      "buildNumber": "5",
+      "buildNumber": "5.3.0",
```

Comment:
It's too late for us to have the build number match the displayed version, but I still think I prefer this to bumping it on every build lol

Reply:
I don't even know if this works

Reply:
It _should_ based on this https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102364